client/src/pages/guide/english/cplusplus/map/index.md

```diff
@@ -55,7 +55,7 @@ Get size of map with size function
 ```
 map<int, int > myMap;
 myMap[100] = 3
-cout << "size of map is " << myMap.size() << '\n';
+count << "size of map is " << myMap.size() << '\n';
```

Comment:
This is a mistake, cout was right. I think it stands for console-out (and similarly cin for reading input).